reaper-adm-extension/src/reaper_adm/exportaction_admsource-earvst.cpp

```diff
@@ -193,8 +193,16 @@ void EarVstExportSources::generateAdmAndChna(ReaperAPI const& api)
                 auto admParameter = (AdmParameter)admParameterIndex;
                 auto param = pluginSuite->getParameterFor(admParameter);
                 auto env = getEnvelopeFor(pluginSuite, pluginInst.get(), admParameter, api);
-
-                if(param && env) {
+                
+                if (getEnvelopeBypassed(env, api)) { 
+                    // We have an envelope, but it is bypassed
```

Comment:
I don't think we can assume there is an envelope at this point - env could be nullptr

Reply:
Looking at `getEnvelopeBypassed` - does `api.GetEnvelopeStateChunk` return false if `env` is `nullptr`? in which case I can see how `getEnvelopeBypassed` would be false if there is no envelope.

Reply:
Thanks for the review. If `env == nullptr`, `getEnvelopeBypassed` returns `false`. But I see that this wasn't a very elegant way to handle it. I've extended the function with an other if-case to handle the  `env == nullptr` case now.